content/en/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -151,10 +151,13 @@ kubectl delete replicaset my-repset --cascade=false
 
 ### Additional note on Deployments
 
-When using cascading deletes with Deployments you *must* use `propagationPolicy: Foreground`
+Prior to 1.7, When using cascading deletes with Deployments you *must* use `propagationPolicy: Foreground`
```

Comment:
I would recommend making the primary documentation indicate to use either Foreground or Background deletion to clean up child replicasets and pods, then add a follow-up note about `Foreground` being required on releases prior to 1.7.

Reply:
Primary documentation already includes a note about cascading deletion cleaning up dependents. Do you want add a special note for Deployments in this section `Additional note on Deployments` ? Something like 
``` 
Use `propagationPolicy: Foreground` or `propagationPolicy: Background` for Deployments to delete both the ReplicaSets created and also their Pods. Prior to 1.7, when using cascading deletes with Deployments you *must* use `propagationPolicy: Foreground` to delete not only the ReplicaSets created, but also their Pods. If this type of _propagationPolicy_ is not used, only the ReplicaSets will be deleted, and the Pods will be orphaned.
```
Does that look good ?

Reply:
in that case, just adding `Prior to 1.7` is probably sufficient. I would drop the specific issue reference.